freshbooks/api/resource.py

```diff
@@ -59,13 +59,13 @@ def _send_request(self, uri: str, method: str, data: Optional[dict] = None) -> A
         if method is HttpVerbs.GET:
             session = self.session.get
         elif method is HttpVerbs.POST:
-            session = self.session.post
+            session = self.session.post  # type: ignore
```

Comment:
Just curious, is the comment "type: ignore" read by mypy or is this just developer information?

Reply:
NVM I see it in the mypy docs :)

Reply:
This tells mypy to ignore the line. 

The mypy update did not like these assignments and the error was fairly intense. Given that this is internal, I figured it was fine to ignore for now:

```
freshbooks/api/resource.py:65: error: Incompatible types in assignment (expression has type "Callable[[Union[str, bytes], Union[None, str, bytes, Mapping[str, Any], Mapping[str, Any], Iterable[Tuple[str, Optional[str]]], IO[Any]], Union[SupportsItems[Union[str, bytes, int, float], Union[str, bytes, int, float, Iterable[Union[str, bytes, int, float]], None]], Tuple[Union[str, bytes, int, float], Union[str, bytes, int, float, Iterable[Union[str, bytes, int, float]], None]], Iterable[Tuple[Union[str, bytes, int, float], Union[str, bytes, int, float, Iterable[Union[str, bytes, int, float]], None]]], str, bytes, None], Optional[Any], Optional[Any], Optional[Any], Optional[Any], Optional[Any], bool, Optional[Any], Optional[Any], Optional[Any], Optional[Any], Optional[Any], Optional[Any]], Response]", variable has type "Callable[[Union[str, bytes], Union[SupportsItems[Union[str, bytes, int, float], Union[str, bytes, int, float, Iterable[Union[str, bytes, int, float]], None]], Tuple[Union[str, bytes, int, float], Union[str, bytes, int, float, Iterable[Union[str, bytes, int, float]], None]], Iterable[Tuple[Union[str, bytes, int, float], Union[str, bytes, int, float, Iterable[Union[str, bytes, int, float]], None]]], str, bytes, None], Optional[Any], Optional[Any], Optional[Any], Optional[Any], Optional[Any], Optional[Any], bool, Optional[Any], Optional[Any], Optional[Any], Optional[Any], Optional[Any], Optional[Any]], Response]")
```